boards/bluepill/Makefile.include

```diff
@@ -9,5 +9,22 @@ PORT_DARWIN ?= $(firstword $(sort $(wildcard /dev/tty.SLAB_USBtoUART*)))
 # setup serial terminal
 include $(RIOTMAKE)/tools/serial.inc.mk
 
-# this board uses openocd
+# optionally, use dfu-util to flash via usb
```

Comment:
Would it make sense to provide a make target for flashing the bootloader? Much like in [1]

[1] https://github.com/RIOT-OS/RIOT/blob/master/makefiles/multislot.mk#L42

Reply:
sounds like a great idea, I'll look into it!